tasks/pr-config.yaml

```diff
@@ -25,7 +28,6 @@ spec:
     - name: registry-secret
     - name: registry-org
     - name: registry-project
-    - name: registry-secret
```

Comment:
Because it is a duplicate:
https://github.com/AICoE/aicoe-ci/blob/master/tasks/pr-config.yaml#L25
https://github.com/AICoE/aicoe-ci/blob/master/tasks/pr-config.yaml#L28